sale_order_project/views/sale_view.xml

```diff
@@ -1,24 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data>
-        <record id="view_order_form" model="ir.ui.view">
-            <field name="name">view_project_quotation_order_form</field>
-            <field name="model">sale.order</field>
-            <field name="inherit_id" ref="sale.view_order_form" />
-            <field name="arch" type="xml">
-                <group name="technical" position="after">
-                    <group name="project" string="Project">
-                        <field name="related_project_id" />
-                    </group>
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sales.order.project.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <group name="technical" position="after">
+                <group name="project" string="Project">
+                    <field name="related_project_id" />
                 </group>
-                <xpath expr="//header/field[@name='state']" position="before">
-                    <button name="action_create_project" type="object"
-                            string="Create Project" class="btn-primary"
-                            groups="project.group_project_user"
-                            attrs="{'invisible':['|',('state','!=','draft'),('related_project_id','!=',False)]}" />
-                </xpath>
-            </field>
-        </record>
-
-    </data>
-</openerp>
+            </group>
+            <xpath expr="//button[@name='action_view_invoice']" position="before">
+                <!-- icon="fa-project-diagram"
+                    This must to be the proper icon, but, until Odoos 12 version the current Font-Awesome version is 4.7, such has not this icon; instead we will use the current icon.
+                    https://github.com/odoo/odoo/issues/21648#issuecomment-353099417
+                -->
+                <button class="oe_stat_button"
```

Comment:
Add a new position for this button. I think it is better to have buttons that creates registers and make changes in the active id on the header. The button box is a nice way to have only buttons to check related models.

Reply:
Well, as you can see `create` button becomes `Project(s) Overview` after to create the new project.

Where must i to put this button?

Reply:
Mmmm, but `crm_project` module use this button in the button box.

Reply:
Can't find crm_project module, is it an OCA one? Could you provide a link?

Reply:
It is in [Odoo](https://github.com/odoo/odoo/tree/11.0/addons/crm_project), and, you can see it in the above `gift`.

Reply:
Ok, I see. You can leave as it is right now if you want, but I still think that the buttons box should only be there for checking related models and activate/deactivate the active model.

Reply:
Ok, we can also get feedback from `OCA` users.

Reply:
Yup, as soon as we merge this my PR will be updated (https://github.com/OCA/project/pull/395) and we'll receive feedback (I guess).